install-php-extensions

```diff
@@ -41,6 +41,22 @@ getDistro () {
 	fi
 }
 
+# Get the distribution name with version
```

Comment:
might be handy to do special case for a specific version of OS

Reply:
Is this function used somewhere?

Reply:
Initially I used this function to switch to libssl1.0-dev in stretch but after that I change to use grep like other conditions for alpine. I will remove the function if you think there is no use for it.